### PR TITLE
workflows: bibclassify task fix

### PR DESCRIPTION
**Pull request by GiorgosPa**

### CODE REVIEW

**Comment by GiorgosPa** [comment]
Pull request updated.

**Comment by jalavik** [comment]
Is this also part of #80 perhaps?

**Comment by GiorgosPa** [comment]
workflows: PEP8 fixes …    97364c7 commit is missing from #80

**Comment by GiorgosPa** [comment]
also    bibclassify: error handling and PEP8 fixes …   684b0aa
is missing

**Comment by jalavik** [comment]
I think also 327b394 is missing from #80.

I added 97364c7 to #80. The bibclassify commit we can keep here, just run `git rebase -i` and keep only the commits you want to add in the rebase list (684b0aa and 327b394). Then you can push and update this PR.

**Comment by GiorgosPa** [comment]
327b394 should replace 2f0aca5. Remove 2f0aca5 from #80 lets keep the rest here.

File: invenio/legacy/bibclassify/ontology_reader.py [hunk 1]
```diff
@@ -769,9 +790,9 @@ def _get_cache(cache_file, source_file=None):
     filestream = open(cache_file, "rb")
     try:
         cached_data = cPickle.load(filestream)
-
-        if cached_data['version_info']['rdflib'] != rdflib.__version__ or \
-                cached_data['version_info']['bibclassify'] != bconfig.VERSION:
+        version_info = cached_data['version_info']
+        if version_info['rdflib'] != (rdflib and rdflib.__version__)\
```
Comment (jalavik) [inline invenio/legacy/bibclassify/ontology_reader.py]: All `rdflib and` can be removed.

File: invenio/legacy/bibclassify/ontology_reader.py [hunk 2]
```diff
@@ -47,9 +47,17 @@
 import traceback
 import xml.sax
 import thread
-import rdflib
+
```
Comment (jalavik) [inline invenio/legacy/bibclassify/ontology_reader.py]: Switch this to only `import rdflib`

**Comment by jalavik** [comment]
Thanks. I missed the comment above. On my machine at the office I have these two missing branches ready to go to `labs`. Just keep this open until then and we'll fix it tomorrow.

**Comment by GiorgosPa** [comment]
Opened #84

**Pull request closed without merging**